### ISSUE TITLE
Elevated circulating malonyl carnitine level

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#11149]
**Preferred term label:**

**Synonyms**
Elevated plasma malonylcarnitine level
Elevated Blood C3DC level
Elevated C3DC in plasma
Elevated levels of malonylcarnitine in blood acylcarnitine profile 

**Definition (free text, please give PubMed ID)**
PMID: 39069445 The diagnosis of MAD is based on elevated levels of malonylcarnitine (C3DC) in the blood acylcarnitine profile and increased excretion of malonic acid on urinary organic acid analysis, which can be accompanied by mildly elevated methylmalonic acid in some patients.

PMID: 37144154 There are elevated levels of malonic acid in urine as well as malonyl carnitine (C3DC) in plasma or whole blood that can be used to diagnose malonic aciduria. Among patients reported, there were 18 (100%, n = 18) patients with urinary malonic acid.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:943 Malonic aciduria

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001315
- label: Elevated circulating malonyl carnitine concentration
- parents: [Abnormal circulating fatty-acid anion concentration (HP:0010966)](https://hpo.jax.org/app/browse/term/HP:0010966)
- pmid: PMID:39069445|PMID:37144154
- def: The concentration of malonylcarnitine in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11149
- synonyms: Elevated plasma malonylcarnitine level(NONE)|Elevated C3DC in plasma(NONE)|Elevated Blood C3DC level(NONE)|Elevated levels of malonylcarnitine in blood acylcarnitine profile(NONE)
